frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.styled.tsx

```diff
@@ -17,6 +18,14 @@ export const ActionSubtitle = styled.span`
   margin-top: 4px;
 `;
 
+export const ActionSubtitlePart = styled.span`
+  &:not(:last-child)::after {
```

Comment:
At least we have a similar code in the other area, so adding a separator in the CSS makes the markup much more simpler. And since we don't want to assert the separator anyway, it'll be fine being in the pseudo element.
https://github.com/metabase/metabase/blob/d502ae762be47dcc1df44080d538b1784e896def/frontend/src/metabase/account/notifications/components/NotificationCard/NotificationCard.styled.jsx#L43-L46

Reply:
Love that!